### ISSUE TITLE
collection feature

### ISSUE THREAD

**Issue by zven** [mint-o-badges/badgr-ui#771]
We used to have the collections feature where users can create collections of badges. 

Old mybadges platform:

![Image](https://github.com/user-attachments/assets/b62e6a97-51d1-4538-97a6-86733ad9a10b)
![Image](https://github.com/user-attachments/assets/793655e0-df8e-4af6-af77-4640feaa4b25)

On the OEB platform we have currently hidden that feature and it needs to be shown again and the design needs to be updated.

**Comment by zven** [comment]
@Annicelia You can use this old issue for the revamped collection feature

**Comment by Annicelia** [comment]
![Image](https://github.com/user-attachments/assets/5dfd64dd-2cf6-475b-b249-488f554f2c91)

**Comment by mint-fini** [comment]
<img width="935" alt="Image" src="https://github.com/user-attachments/assets/5968ad08-f0d7-444d-a6b6-22d99575476e" />

I can create collections! But I did not find the "Public" button during my testing, and the "Share" button is greyed out and does not work.

**Comment by zven** [comment]
@mint-fini Yes, that's the intended behaviour for now! We are going to implement the sharing feature in the next sprint.
